src/com/xilinx/rapidwright/device/IntentCode.java

```diff
@@ -160,6 +160,11 @@ public boolean isUltraScaleClockDistribution(){
     private static final int ULTRASCALEPLUS_START_IDX = ULTRASCALE_START_IDX;
     private static final int ULTRASCALEPLUS_END_IDX = ULTRASCALE_END_IDX + 4;
 
+    private static IntentCode[] values = values();
```

Comment:
A bit of a mockery of name overloading, but perhaps this is more elegant?
```suggestion
    public final static IntentCode[] values = values();
```